src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -224,6 +221,22 @@ trait ClusterComponent extends LeoComponent {
         .map { recs => recs.headOption.flatten.map(ServiceAccountKeyId) }
     }
 
```

Comment:
Moved up the `get*` methods below so `get*/set*/etc.` methods are together for easier finding.